File: java-sandbox/java-maven-tests/src/main/java/com/example/sandbox/Calculator.java
package com.example.sandbox;

/** Tiny "system under test" used by the example tests. */
public final class Calculator {

    private Calculator() {
    }

    public static int add(int a, int b) {
        return a + b;
    }

    public static int divide(int dividend, int divisor) {
        if (divisor == 0) {
            throw new IllegalArgumentException("division by zero");
        }
        return dividend / divisor;
    }

    public static boolean isEven(int n) {
        return n % 2 == 0;
    }

    public static String fizzbuzz(int n) {
        if (n % 15 == 0) {
            return "FizzBuzz";
        }
        if (n % 3 == 0) {
            return "Fizz";
        }
        if (n % 5 == 0) {
            return "Buzz";
        }
        return String.valueOf(n);
    }
}

File: csharp-sandbox/cs-mstest-tests/src/Calculator.cs
namespace Sandbox;

/// <summary>Tiny "system under test" used by the example tests.</summary>
public static class Calculator
{
    public static int Add(int a, int b) => a + b;

    public static int Divide(int dividend, int divisor)
    {
        if (divisor == 0)
        {
            throw new ArgumentException("division by zero", nameof(divisor));
        }

        return dividend / divisor;
    }

    public static bool IsEven(int n) => n % 2 == 0;

    public static string FizzBuzz(int n) => n switch
    {
        _ when n % 15 == 0 => "FizzBuzz",
        _ when n % 3 == 0 => "Fizz",
        _ when n % 5 == 0 => "Buzz",
        _ => n.ToString(),
    };
}
